src/cmd/prepare.ts

```diff
@@ -106,12 +109,14 @@ export async function prepare({
   const releaseDescription = config.user.getReleaseDescription
     ? await config.user.getReleaseDescription(hookCtx)
     : `This PR was opened by the ` +
-      `[ready-release-go](https://github.com/woodpecker-ci/plugin-ready-release-go) plugin.` +
+      `[ready-release-go](https://github.com/woodpecker-ci/plugin-ready-release-go) plugin. ` +
       `When you're ready to do a release, you can merge this and a release and tag with ` +
-      `version \`${nextVersion}\` will be created automatically.` +
+      `version \`${nextVersion}\` will be created automatically. ` +
       `If you're not ready to do a release yet, that's fine, ` +
-      `whenever you add more changes to \`${releaseBranch}\`` +
+      `whenever you add more changes to \`${releaseBranch}\` ` +
       `this PR will be updated.\n\n` +
+      `## Options\n\n` +
+      `- [${shouldBeRC ? "x" : " "}] Release this version as RC\n\n` +
```

Comment:
```suggestion
      `- [${shouldBeRC ? "x" : " "}] Mark this version as a "Release Candidate" (RC)\n\n` +
```

Reply:
adjusted the text (just without `(RC)`) to keep the regex simple.